tests/docker-compose-storage.yml

```diff
@@ -0,0 +1,31 @@
+services:
+  minio:
+    image: minio/minio:RELEASE.2023-09-20T22-49-55Z
+    command: server /data --console-address ":9001"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+      interval: 5s
+      timeout: 5s
+      retries: 10
+    ports:
+      - "9000:9000"
+      - "9001:9001"
+    environment:
+      MINIO_ROOT_USER: distribution
+      MINIO_ROOT_PASSWORD: password
+    volumes:
+      - ./miniodata/distribution:/data
+
+  minio-init:
+    image: minio/mc:RELEASE.2023-02-16T19-20-11Z
+    depends_on:
+      minio:
+        condition: service_healthy
+    entrypoint: >
+      /bin/bash -c "
+        /usr/bin/mc config host add minio http://minio:9000 $${MINIO_ROOT_USER} $${MINIO_ROOT_PASSWORD} && (
+          /usr/bin/mc stat minio/images-local || /usr/bin/mc mb minio/images-local
+        ) && /usr/bin/mc anonymous set public minio/images-local && /usr/bin/mc admin trace --path "
```

Comment:
I had trouble here with not specifying the path argument so took out `&& /usr/bin/mc admin trace --path` - I'm not sure what it was supposed to be adding.

Reply:
Yes thats my copy-pasta 😅 trace shouldnt be there I dont think

Reply:
Should be fixed here https://github.com/distribution/distribution/pull/4069/commits/14361b3ab5f5b724f2ee153ab9fa697a62cd86dc